WooCommerce/WooCommerce.xcodeproj/xcshareddata/xcschemes/WooCommerce.xcscheme

```diff
@@ -129,7 +129,7 @@
          </CommandLineArgument>
          <CommandLineArgument
             argument = "-simulate-stripe-card-reader"
-            isEnabled = "NO">
+            isEnabled = "YES">
```

Comment:
Will omit this change when I publish the PR (i.e. as not draft)

Reply:
Removed in https://github.com/woocommerce/woocommerce-ios/pull/4867/commits/76fc831930b7646dbad863ba0a93ccea14efd7b2